### PR TITLE
fixes #18277 - adds a metadata file to backup

### PR DESCRIPTION
**Pull request by cfouant**

### CODE REVIEW

**Review by chris1984 — APPROVED** [review]
Looks good to me

File: katello/katello-backup [hunk 1]
```diff
@@ -123,6 +123,16 @@ else
     '/var/lib/puppet/ssl'
   ]
 
+  puts "Generating metadata ... "
+  metadata = File.new 'metadata', 'w'
+  metadata.puts `cat /etc/redhat-release`
+  metadata.puts "Plugin list: "
+  plugin_list = `foreman-rake plugin:list | grep 'Foreman plugin: '`.lines
```
Comment (chris1984) [inline katello/katello-backup:130]: I will have to save that command for my records, awesome this exists.

**Comment by stbenjam** [comment]
Commit message needs a # before the issue number, otherwise :+1:

File: katello/katello-backup (hunk 1 above)
Comment (cfouant) [reply katello/katello-backup:130]: Right? I looked for so long to find a way... @stbenjam was the man with the answers! Hero of my week!

**Comment by cfouant** [comment]
@stbenjam - fixed the commit message, thanks!

**Review by stbenjam — APPROVED** [review]
LGTM

**Comment by ehelms** [comment]
I think it would have been nice if this was a known structured file type (unless I'm missing something). This appears to be free form vs something like YAML or JSON that could be parsed by automation easier.

**Pull request merged**